app/page.tsx

```diff
@@ -1,48 +1,205 @@
+                    fill="#93C9FE"
+                  />
+                  <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M19.7875 11.2457C19.8303 10.2575 19.9862 9.49598 20.1229 8.84188C20.3293 7.85482 20.4915 7.46558 20.8938 6.05322C21.0532 5.49362 21.2702 4.70365 21.5086 3.73592C21.5516 3.56138 21.4594 3.34425 21.3038 3.25428C19.4587 2.18734 17.6136 1.12049 15.7684 0.0536447C15.6001 -0.0436424 15.3562 -0.00488437 15.2263 0.139783L12.9306 2.69851C13.2272 3.01032 13.483 3.26734 13.8703 3.61364C14.0372 3.76292 14.2139 3.92039 14.4061 4.08344C14.9017 3.53325 15.3972 2.98306 15.8928 2.43287L19.3141 4.4232C18.9404 5.4256 18.6733 6.28507 18.4878 6.93298C18.2061 7.91718 18.0119 8.6055 17.8736 9.57219C17.7325 10.5586 17.7226 11.3244 17.722 11.6069C17.7208 12.3816 17.7841 13.4349 18.0564 14.6655C18.7067 14.4277 19.2802 14.2751 20.0021 14.0936C19.7806 12.9476 19.7562 11.9715 19.7875 11.2457Z"
+                    fill="#0078E8"
+                  />
+                </svg>
+              </span>
+              <span className={styles.title}>blueprint </span>
+              <span className={styles.blog}>blog</span>
+            </div>
+          </div>
+        </div>
+
+        <div className={styles.profile}>
+          <div className={styles.picName}>
+            <div className={styles.avatar} />
+            <span className={styles.name}>rbeggs</span>
+          </div>
+          <span className={styles.date}>September 19</span>
+        </div>
+        <div className={styles.words}>
           In response to the growing homelessness crisis in San Francisco, a
           local nonprofit organization, Code Tenderloin, has launched a
           comprehensive initiative aimed at providing long-term solutions for
           individuals experiencing homelessness. The organization, founded in
           2015, is dedicated to addressing both immediate needs and underlying
           causes of homelessness through a combination of shelter services, job
           training programs, and mental health support. Read more online:
-          https://www.codetenderloin.org/
-        </p>
-
-        <p>
-          Image Link:
-          https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg
-        </p>
-
-        <HeartIcon size={24} />
-        <p>256 Likes</p>
-        <ShareIcon size={24} />
+          https:// www.codetenderloin.org/
+        </div>
+        <div className={styles.pic}></div>
+        <span className={styles.withLine}>
+          <span className={styles.picName}>
+            <HeartIcon size={24} />
+            <span className={styles.name}>256 Likes</span>
+          </span>
+          <ShareIcon size={24} />
+        </span>
 
-        <ProfileIcon size={24} />
-        <p>daviddd</p>
-        <p>September 20</p>
-        <p>
+        <div className={styles.profile}>
+          <div className={styles.picName}>
+            <div className={styles.avatar} />
```

Comment:
Avoid using empty divs for shapes. You should be using the `<ProfileIcon size={24} />`, imported from react-icons.